src/db/migrate.rs

```diff
@@ -286,11 +286,16 @@ pub fn migrate(version: Option<Version>, conn: &mut Client) -> crate::error::Res
             // version
             7,
             // description
-            "Allow memory limits of more than 4 GB",
+            "Allow memory limits of more than 2 GB",
             // upgrade query
             "ALTER TABLE sandbox_overrides ALTER COLUMN max_memory_bytes TYPE BIGINT;",
             // downgrade query
-            "ALTER TABLE sandbox_overrides ALTER COLUMN max_memory_bytes TYPE INTEGER;"
+            "
+                ALTER TABLE sandbox_overrides
+                    ALTER COLUMN max_memory_bytes
+                    TYPE INTEGER
+                    USING least(max_memory_bytes, 2147483647);
```

Comment:
why this change? 

In the end, it would only be executed on new databases,

Reply:
The tests run through all the downgrades, so since I added a test that applied an override of >2GB this started failing.

Reply:
(It might be worth removing old migrations at some point and merging them into a higher base database schema).